packages/node/src/algorand/api.service.algorand.ts

```diff
@@ -43,11 +48,16 @@ export class AlgorandApiService extends ApiService {
       ? network.endpoint
       : [network.endpoint];
 
-    const connections = await Promise.all(
+    const endpointToApiIndex: Record<string, AlgorandApiConnection> = {};
+
+    await Promise.all(
```

Comment:
If any of these fail indexing wont start up. We should continue to run even if an endpoint doesn't work